agent/elia_chat/screens/help_screen.py

```diff
@@ -0,0 +1,165 @@
+from typing import ClassVar
+from typing import List
+
+from textual.app import ComposeResult
+from textual.binding import Binding
+from textual.containers import Vertical
+from textual.containers import VerticalScroll
+from textual.screen import ModalScreen
+from textual.widgets import Footer
+from textual.widgets import Markdown
+
+
+class HelpScreen(ModalScreen[None]):
+    BINDINGS: ClassVar[List[Binding]] = [
+        Binding("q", "app.quit", "Quit", show=False),
+        Binding("escape,f1,?", "app.pop_screen()", "Close help", key_display="esc"),
+    ]
+
+    HELP_MARKDOWN = """\
+### How do I quit Elia?
+
+Press `Ctrl+C` on your keyboard.
+`q` also works if an input isn't currently focused.
+If focus is on the prompt input box on the home screen, `esc` will close Elia too.
+
+### Environment
+
+You may need to set some environment variables, depending on the model
+you wish to set.
+To use OpenAI models, the `OPENAI_API_KEY` env var must be set.
+To use Anthropic models, the `ANTHROPIC_API_KEY` env var must be set.
+
+To use a local model, see the instructions in the README:
+
+* https://github.com/darrenburns/elia/blob/main/README.md
+
+### Config file and database
+
+The locations of the config file and the database can be found at the bottom
+of the options screen (`ctrl+o`).
+
+### General navigation
+
+Elia has very strong mouse support. Most things can be clicked.
+
+Use `tab` and `shift+tab` to move between different widgets on screen.
+
+In some places you can make use of the arrow keys or Vim nav keys to move around.
+
+In general, pressing `esc` will move you "closer to home".
+Pay attention to the bar at the bottom to see where `esc` will take you.
+
+If you can see a scrollbar, `pageup`, `pagedown`, `home`, and `end` can also
+be used to navigate.
+
+On the chat screen, pressing `up` and `down` will navigate through messages,
+but if you just wish to scroll a little, you can use `shift+up` and `shift+down`.
+
+### The chat list
+
+- `up,down,k,j`: Navigate through chats.
+- `a`: Archive the highlighted chat.
+- `pageup,pagedown`: Up/down a page.
+- `home,end`: Go to first/last chat.
+- `g,G`: Go to first/last chat.
+- `enter,l`: Open chat.
+
+### The options window
+
+Press `ctrl+o` to open the _options window_.
+
+On this window you can change the `model` and `system prompt`.
+The system prompt tells the model to behave, and is added to the
+start of every conversation.
+
+Changes made on the options window are saved automatically.
+
+> **`Note`**: Changes made in the options window only apply to the current session!
+
+You can change the system prompt globally by updating the config file.
+The location of the config file is shown at the bottom of the options window.
+
+### Writing a prompt
+
+The shortcuts below work when the _prompt editor_ is focused.
+The prompt editor is the box where you type your message.
+It's present on both the home screen and the chat page.
+
+- `ctrl+j`: Submit the prompt
```

Comment:
Let's switch submit and and newline - enter to submit, ctrl+j for multiline input

Reply:
@jssmith 
1) "newline - enter to submit": Are you suggesting that you have to hit enter twice (first to create a new line, then a second time to submet)?
2) "enter to submit": that we submit when you hit enter? If so, it won't be possible to do multiline unless you somehow hit a key to switch from single line to multiline -- is that what you mean?